sky/packages/updater/lib/main.dart

```diff
@@ -67,14 +100,15 @@ class UpdateTask {
   String _dataDir;
   String _tempPath;
 
-  _readLocalManifest() async {
-    String manifestPath = path.join(_dataDir, kManifestFile);
-    String manifestData = await new File(manifestPath).readAsString();
-    _currentManifest = yaml.loadYaml(manifestData, sourceUrl: manifestPath);
+  Future _readLocalManifest() async {
+    String bundlePath = path.join(_dataDir, kBundleFile);
```

Comment:
Is there a performance benefit to doing that? Otherwise it seems overly verbose to do this for short-lived local variables.